chat-client/src/client/messager.ts

```diff
@@ -109,9 +110,6 @@ export class Messager {
     }
 
     onError = (params: ErrorParams): void => {
-        this.chatApi.tabIdReceived({
-            eventId: params.eventId || '',
-            tabId: params.tabId,
-        })
+        this.chatApi.telemetry(params)
```

Comment:
The same - `ErrorParams` does not contain `name`? Do you plan to convert telemetry params to proper entry on server side? I'd say it needs to be properly defined here.

Reply:
I didn't want to focus on telemetry for this PR, the telemetry related changes in this PR are a byproduct of the version bump in chat-ui-types, telemetry will be separate PR